### ISSUE TITLE
Compilation warning on collectionView(_:layout:mozaikSizeForItemAtIndexPath:)

### ISSUE THREAD

**Issue by cyrilchandelier** [Antondomashnev/ADMozaicCollectionViewLayout#15]
Since a couple versions of Xcode (probably after migration to Swift 3.0), I have this warning that I can not silent:

```
Instance method 'collectionView(_:layout:mozaikSizeForItemAtIndexPath:)' nearly matches optional requirement 'collectionView(_:layout:sizeForItemAt:)' of protocol 'UICollectionViewDelegateFlowLayout'
```

Any idea how to fix that?

Thanks,
Cyril

**Comment by Antondomashnev** [comment]
Hi @cyrilchandelier, thanks for reporting it. I'll check it and get back to you soon 😄

**Comment by Antondomashnev** [comment]
Hi @cyrilchandelier, the fix is merged and the new 3.0.0 release is out 🚀

**Comment by cyrilchandelier** [comment]
Thank you for being this reactive!
